tensorflow/core/common_runtime/process_util.cc

```diff
@@ -97,18 +97,39 @@ int32 NumIntraOpThreadsFromEnvironment() {
   return (val && strings::safe_strto32(val, &num)) ? num : 0;
 }
 
+#ifdef INTEL_MKL
+int32 OMPThreadsFromEnvironment() {
+  int32 num;
+  const char* val = std::getenv("OMP_NUM_THREADS");
```

Comment:
Please add a comment saying that 
1) `std::getenv` is thread-safe (as long as no other function modifies the host environment) from C++11 onward.
2) Most of TF code (except tests and experimental code) doesn't call `setenv` and `unsetenv`.